### ISSUE TITLE
Maybe worth chatting to these guys as some overlap on the meta-analysis side of things

### ISSUE THREAD

**Issue by DrMattG** [mcguinlu/ESinR#9]
Harrer, M., Cuijpers, P., Furukawa, T.A, & Ebert, D. D. (2019). Doing Meta-Analysis in R: A Hands-on Guide. DOI: 10.5281/zenodo.2551803. https://bookdown.org/MathiasHarrer/Doing_Meta_Analysis_in_R/

**Comment by mcguinlu** [comment]
Yeah, so I contributed to that guide (Risk of bias plots) - it's currently undergoing editing, to be published at the end of 2020.

My initial thinking for this guide was that it would deal with wider evidence synthesis issues than just the analysis (e.g. information retrival, reporting, full text retrieval, management of workflows, etc). See this tweet from the intial thread on this guide:

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">It would deal with the other side of SRMAs to the wonderful &quot;Doing Meta-Analysis with R&quot; by <a href="https://twitter.com/MathiasHarrer?ref_src=twsrc%5Etfw">@MathiasHarrer</a> and co, which is focused on the actually analysis<a href="https://t.co/c5Z9iHFkTT">https://t.co/c5Z9iHFkTT</a></p>&mdash; Luke McGuinness (@mcguinlu) <a href="https://twitter.com/mcguinlu/status/1391705845835128839?ref_src=twsrc%5Etfw">May 10, 2021</a></blockquote> 

However, most people so far have proposed very analytical chapters, so I am not 100% sure what to do. Comments/suggestions/thoughts on this appreciated (hence the label)!

**Comment by mcguinlu** [comment]
Actually, also tagging @MathiasHarrer (main author on the other guide) for his input here, as well as @nealhaddaway.

**Comment by DrMattG** [comment]
The gap in the "market" is, I think, as you articulated - the wider ES issues.  However, there seems to be an imbalance in the treatment of analytical ES approaches - meta-analysis in R has been done really well but some of the other things have not been done (or are too focused in one discipline - and usually the one I am not in!) . Would that be sufficent justification to, for meta-analysis specifically, point to other very useful resources (Mathias's book and Wolfgangs examples from metafor for example)? We could then still keep some of the other analytical chapters in without seeming unbalanced. (I need more coffee this morning... I hope this rambling makes some sense)

**Comment by MathiasHarrer** [comment]
Hey guys, thanks for tagging me! You raise several very goods points here I think. 

The "Doing Meta" guide focuses, in its own humble ways, on the actual analysis steps in R. But I often get the feeling that there are so many important issues that have nothing to do with running the MA itself (at least not directly). Many of these things happen long before the meta-analysis is run, and they can cause MA beginners to struggle with their MA. 

Here's a few things that have come up many times in our experience (based on teaching students, feedback on the guide, etc.):

- **Problem Specification and Research Questions**: especially with SMR/MAs, it is often difficult to define precisely what it should achieve. What "problem" does it solve, and for whom? This point is so important because many other things (dealing with heterogeneity, moderator analyses, and the general analysis plan) depend on it. I know this has nothing to do with R in particular. But especially for complete beginners, it could be so helpful to get some guidance on how to define a good SMR/MA research question (e.g. PICO, FINER, other good approaches, how to avoid redundant SMR/MAs, which are abundant at least in my field).
- **Search, Information Retrieval etc.**. @mcguinlu you mentioned this point. For beginners, it is hard to understand how to conduct an SMR that fulfills certain quality standards, and this aspect can be overwhelming. Having a practical, low-threshold guide on this would be wonderful, especially if it also discussed R tools, PRISMA/MARS guidelines etc. 
- **Collecting an structuring data**. This is yet another point that is often taken for granted, but often causes problems. How do i collect (complex) SMR/MA data (RoB assessments, raw ES data, moderator data, multiple outcomes per study) in connection with R and other tools?
- **Effect sizes**. This is definitely a huge part @DrMattG. How do i calculate this or that effect size from raw data using R? How can i calculate many ES at the same time? How do i deal with dependent ES? What is the sampling variance/SE, of an effect size, and why do we need it? One could introduce escalc here for example. The new fully revised version of "Doing Meta" ( will also deal with ES and ES calculation in a new chapter, but i got the feeling that we could only scratch the surface.

In terms of redundancy, I think you do not have worry _too_ much. I think every piece of information on the web is valuable. I would also be more than happy to link to the new guide in "Doing Meta" for things we do not cover, or simply when the new guide explains things better than we could. Or in more detail. The goal i think should be to create an infrastructure that makes it as easy as possible for beginners to get the information they need for conducting a high quality SMR/MA using R.

Sry for the verbose comment ;)

@mcguinlu I'll send you the full version of the revised guide (currently in production) asap, so you can see what will change. ;)

**Comment by mcguinlu** [comment]
@DrMattG I agree completely - hopefully we can describe a lot of these alternative methods in #11 (maybe you could liase with @nealhaddaway). The main method in my field is meta-analysis, so I'm keen to get alternative viewpoints!

And @MathiasHarrer thanks for commenting here - really useful to get your input! 

So for the four points you mentioned:
- **Problem Specification and Research Questions**: I agree - while it has very little to do with R per say, it would be good to help frame the issues for users, so that people completely new to ES/MA don't just jump in the deep-end. This is particularly important, as you say, because descisions made early on really affect subsequent issues.
- **Search, Information Retrieval etc**: This should be covered by #13
- **Collecting an structuring data**: I think this could be a key area where this guide comes into its own, and we could definitely consider things like automated extraction here . And then also expanded topics, like using unique IDs to link between the summary table data, effect estimates and ROB assessments for the same result, or workflows to highligh how to move data from one stage of the exercise to the next. I'll open an issue on this momentarily.
- **Effect sizes**: This is a great point, and something we can definitely touch on! I'll open an issue on this momentarily.


> The goal i think should be to create an infrastructure that makes it as easy as possible for beginners to get the information they need for conducting a high quality SMR/MA using R.

Couldn't agree more! Wish something like this existed when I started 😂

**Comment by nealhaddaway** [comment]
I'm wondering whether anyone would use R for qualitative synthesis methods like thematic synthesis. In theory there are some cool functions for highlighting text within a corpus and codes and codebooks can be easily and transparently generated, but I'm not sure it's in common use for that kind of thing...

**Comment by mcguinlu** [comment]
Great point @nealhaddaway!

> I'm wondering whether anyone would use R for qualitative synthesis methods like thematic synthesis. In theory there are some cool functions for highlighting text within a corpus and codes and codebooks can be easily and transparently generated, but I'm not sure it's in common use for that kind of thing...

Have [tweeted your question about this](1394661665782837250), so hopefully someone might know!
